lex.go

```diff
@@ -159,7 +159,6 @@ WriteToBuf:
 // lexSymbolState scans for strings and can emit the following tokens:
 // orToken, andToken, containsToken
 func lexSymbolState(l *lexer) (lexStateFunc, error) {
-	const op = "mql.lexSymbolState"
 	panicIfNil(l, "lexSymbolState", "lexer")
```

Comment:
I'd prefer that you leave `op` and just do:
```suggestion
	panicIfNil(l, op, "lexer")
```

Reply:
Why not disable the rule instead? 

```golang
//nolint:unused
const op = "mql.lexSymbolState"
```

And since `panicIfNil(l, "lexSymbolState", "lexer")` is a no-op, let's remove it as well?

Reply:
How is `panicIfNil(...)` a no op?

No reason to disable the rule: just use the `op` pattern along with `panicIfNil(...)` to express the caller?

Reply:
This change has been made.